src/lib/group.go

```diff
@@ -343,6 +343,76 @@ func group_add(ctx pf.PfCtx, args []string) (err error) {
 	return
 }
 
+//Function to allow TG admins and sysadmins to request a reset of the passowrd
+//for a member of a group that they posses an admin bit for.
```

Comment:
I normally see / use // comments as:
// foo bar baz

not:
//foo bar baz

It looks like most other comments are /* */ style, so if you want to keep these as:
//foo bar baz

that seems ok to me.

There is a mis-spelling:
  possess
two s's at the end.